action/generate.go

```diff
@@ -54,8 +58,14 @@ func (s *Action) Generate(ctx context.Context, c *cli.Context) error {
 	}
 
 	if length == "" {
-		length = strconv.Itoa(defaultLength)
-		if l, err := s.askForInt("How long should the password be?", defaultLength); err == nil {
+		def := defaultLength
```

Comment:
Why is this variable called `def`? What does it mean? Can you use a more descriptive name? `candidateLength` or `lengthAsString` perhaps, if that is what it represents here?

Reply:
At the second look this whole length calculation block looks a little odd to me. First converting from int to string and later back to int doesn't feel right.

Maybe you could untangle this a bit?